src/MultiplexingDistributedCache/MultiplexingDistributedCache.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Aqovia.Cache</RootNamespace>
+    <AssemblyName>Aqovia.Cache.MultiplexingDistributedCache</AssemblyName>
+    <PackageId>Aqovia.MultiplexingDistributedCache</PackageId>
+    <VersionPrefix>0.1.0</VersionPrefix>
+    <Authors>Aqovia</Authors>
```

Comment:
Nice to have: I wonder if we could populate this from Github commit history :-) Definitely not for today, I just wanted to write the idea down!

Reply:
I thought the Authors is the name of the company, but I can see For [StackExchange.Redis](https://github.com/StackExchange/StackExchange.Redis/blob/ae6419a164600b4b54dd7ce8a699efe8e98d8f1c/Directory.Build.props#L9), it's:

```xml
    <Authors>Stack Exchange, Inc.; Marc Gravell; Nick Craver</Authors>
```

It can be done in that common github action for versioning, so it updates a line based on a regular expression.